lamindb/dev/datasets/_core.py

```diff
@@ -132,6 +132,17 @@ def anndata_pbmc68k_reduced() -> ad.AnnData:
     return ad.read(filepath)
 
 
+def anndata_file_pbmc68k_test() -> Path:
+    """Modified from scanpy.datasets.pbmc68k_reduced().
+
+    Additional slots were added for testing purposes.
```

Comment:
So far, for each test asset, we've added the exact steps for exact reproducibility of the dataset. Could you please just add the lines of code that were being run similar to how it's done for the other datasets?

Reply:
will do.

Reply:
Thanks!